docs/OV_Runtime_UG/layout_overview.md

```diff
@@ -1,24 +1,24 @@
-# Layout API overview {#openvino_docs_OV_UG_Layout_Overview}
+# Layout API Overview {#openvino_docs_OV_UG_Layout_Overview}
```

Comment:
This article has been reviewed in the Transition Guide. An updated version is in master. Please fetch and update it first. @sgolebiewski-intel